### ISSUE TITLE
Pretty sure thuamcraft is responsible at this point my game actually crashed after i built the infusion altar

### ISSUE THREAD

**Issue by Coletrain927** [talonos2/BlightfallTracker#211]
The report was not generated as my computer completely froze and i ended up restarting as i could not even bring up the task manager so i have no hard evidence i would like some to look into it. I love the blightfall modpack but this is a persistent issue as i cannot progress past this point i have no other problems with the modpack or with any other modpacks. I cannot get a report becausemy game does not crash butit seems that it starts to lag as soon as start creating objects with thuamcraft usually goes to hell(excuse my french) after the infusion altar is built. I am completely frustrated cause i have to keep restarting help!

**Comment by talonos2** [comment]
The problem is, we cannot reproduce it on our end, and you are not giving
us enough info to diagnose the problem. I appreciate that you've narrowed
it down to the infusion altar, but I and all my testers have built infusion
altars in our games and haven't seen this occur.

What is your video card,OS,  CPU, and RAM situation? Do you have the most
up to date drivers? What version of java are you running?

If you play with the console up, is there any console spam?

If you start a new world, switch to creative mode, and build an infusion
altar there, does it seem to lag? Does it lag when you create an infusion
altar in other modpacks?

Can we get screenshots of the base?

If worst comes to worst, could you copy your world save folder, zip it up,
and put it on drop-box or something so I can have a look at it?

We cannot "look into" something that doesn't happen on our machines. I have
never seen anybody with your problem before. Right now, your situation is
just a persistent anecdote. I'd love to help, but we need more info.

**Comment by Coletrain927** [comment]
I have an intel(R) Core(TM) i7 cpu with 6.00 GB RAM
920 @2.67 GHz 2.66 GHz with 64-bit OS my java is update with run time environment 8 as for the world do you have an email?

**Comment by talonos2** [comment]
Yes, but not one that can accept a direct file of that size. Just upload
the world somewhere and link it here.

Your OS is 64 bit, but what is it, exactly? Linux? Windows? Mac? And what's
your graphics card situation?

Were you using a builder's wand by any chance? I _just_ received a report
from somebody else that the builder's wand, when used with Thaumcraft, can
cause severe lag permanently afterwards in a chunk.

**Comment by Coletrain927** [comment]
My OS is windows and i don't recall using a builders wand on the structure due to the detail but i might have used it with creating the cobble stone foundation for the structure and i should mention that i several pure nodes buried beneath the structure to prevent taint from engulfing it. could they have a effect on the altar? i know thuamcraft has a mechanism that interacts the world around it. In anycase im uploading the world now

**Comment by Coletrain927** [comment]
Oh and it's windows 7 vista

**Comment by Raclex** [comment]
I have big lag with centivis nodes on my screen.  Maybe something similar is happening with these nodes you mentioned. Do you lag only around the nodes?
